blogs/lokiless/lokiless.md

```diff
@@ -0,0 +1,282 @@
+
+```bash
+# Create a namespace for all the deployments
+kubectl create namespace netobserv
+
+# Install Strimzi (Kafka)
+kubectl apply -f https://strimzi.io/install/latest?namespace=netobserv -n netobserv
+export DEFAULT_SC=$(kubectl get storageclass -o=jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubernetes\.io/is-default-class=="true")].metadata.name}') && echo "Using SC $DEFAULT_SC"
+curl -s -L "https://raw.githubusercontent.com/jotak/kafka-clickhouse-example/main/contrib/kafka.yaml" | envsubst | kubectl apply -n netobserv -f -
+
+# Wait that all pods are up and running, with the KafkaTopic being ready (a few minutes...)
+kubectl wait --timeout=180s --for=condition=ready kafkatopic flows-export -n netobserv
+kubectl get pods -n netobserv
+```
+
+When listing pods, you should see something like this:
+
+```
+NAME                                             READY   STATUS    RESTARTS   AGE
+kafka-cluster-entity-operator-5f5bc5675b-lpmmc   3/3     Running   0          3m33s
+kafka-cluster-kafka-0                            1/1     Running   0          4m47s
+kafka-cluster-kafka-1                            1/1     Running   0          4m47s
+kafka-cluster-kafka-2                            1/1     Running   0          4m47s
+kafka-cluster-zookeeper-0                        1/1     Running   0          5m15s
+kafka-cluster-zookeeper-1                        1/1     Running   0          5m15s
+kafka-cluster-zookeeper-2                        1/1     Running   0          5m15s
+strimzi-cluster-operator-6bdcd796f6-qsc4g        1/1     Running   0          17m
+```
+
+### Prepare NetObserv
+
+Assuming you already installed the operator, now you must create a `FlowCollector` resource that will start sending flow logs to Kafka. For the purpose of this article, we won't install and configure Loki but be informed that you can export flows to one or multiple endpoints and save them into Loki in parallel according to your needs.
+
+> Note: we configure here Kafka as an **exporter**, which is unrelated to the `spec.deploymentModel: KAFKA` / `spec.kafka` settings: those ones correspond to NetObserv's internal flows processing configuration (NetObserv being both the producer and the consumer), whereas `spec.exporters` relates to NetObserv being just the producer, leaving up to us how we want to consume that data.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: flows.netobserv.io/v1beta1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  loki:
+    enable: false
+  exporters:
+    - type: KAFKA
+      kafka:
+        address: "kafka-cluster-kafka-bootstrap.netobserv"
+        topic: flows-export
+EOF
+```
+
+The Kafka `address` and `topic` configured here match what we deployed in the previous step.
+
+At this point, flows should be generated, collected, enriched and sent to Kafka, but with no consumer on the other end. 
```

Comment:
You don't need a comma before this "but" because "with no consumer on the other end" is not an independent clause w/ a subject and verb. A comma is only needed before a conjunction if there's an independent clause on the other side.